README.md

```diff
@@ -18,6 +18,7 @@ To learn how to get started with Cloud Functions for Firebase by having a look a
 This repository contains a few categories of samples:
 
  1. [**Quickstarts**](#quickstarts) are minimal examples for each types of triggers.
+ 1. [**Cloud Functions v2 public preview**](#v2) samples show how to use the public preview of [Cloud Functions for Firebase v2](https://firebase.google.com/docs/functions/beta).
  1. [**Development Environment Samples and Boilerplates**](#environment) illustrates how to get started with
 different, commonly used JavaScript development patterns such as Typescript, React SSR, ES2017 etc...
```

Comment:
Suggest "SSR, and ES2017."

I'd do the same with the next two ellipses below.  "Such as" lets you give a few examples, but suggests more.